com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/ProcessLoggerProviderFactory.java

```diff
@@ -1,212 +0,0 @@
-package com.sap.cloud.lm.sl.cf.persistence.services;
-
-import static java.text.MessageFormat.format;
-
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
-
-import javax.inject.Inject;
-
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
-import org.apache.log4j.PatternLayout;
-import org.apache.log4j.spi.NOPLogger;
-import org.apache.log4j.varia.NullAppender;
-import org.flowable.engine.delegate.DelegateExecution;
-import org.slf4j.LoggerFactory;
-
-import com.sap.cloud.lm.sl.cf.persistence.message.Constants;
-import com.sap.cloud.lm.sl.cf.persistence.message.Messages;
-
-public class ProcessLoggerProviderFactory {
```

Comment:
https://memegenerator.net/img/instances/62582777.jpg

May the lord (Dido Hristov) have mercy on your soul.